src/social-network/worker.ts

```diff
@@ -25,23 +25,23 @@ export interface SocialNetworkWorkerDefinition extends SocialNetworkWorkerAndUI
      *
      * If this function is not provided, autoVerifyBio in Welcome will be unavailable
      */
-    autoVerifyBio?(user: PersonIdentifier, provePost: string): void
+    autoVerifyBio?: ((user: PersonIdentifier, provePost: string) => void) | null
     /**
      * This function should open a new page, then automatically input provePost to the post box
      *
      * If this function is not provided, autoVerifyPost in Welcome will be unavailable
      */
-    autoVerifyPost?(user: PersonIdentifier, provePost: string): void
+    autoVerifyPost?: ((user: PersonIdentifier, provePost: string) => void) | null
     /**
      * This function should open a new page, then let user add it by themself
      *
      * If this function is not provided, manualVerifyPost in Welcome will be unavailable
      */
-    manualVerifyPost?(user: PersonIdentifier, provePost: string): void
+    manualVerifyPost?: ((user: PersonIdentifier, provePost: string) => void) | null
 }
 
 export type SocialNetworkWorker = Required<SocialNetworkWorkerDefinition>
-export const getActivatedWorker = getCurrentNetworkWorker
+export const getWorker = getCurrentNetworkWorker
```

Comment:
So why not use getCurrentNetworkWorker? It's name is clearer

Reply:
No "current", just ```getNetworkWorker```. If you thought that ```getWorker``` is too short, this name could be better. Please let me know if this name is okay.

Reply:
Or maybe getSocialNetworkProviderWorkerByIdentifier, that's more precise ah whatever don't let the name be too simple

Reply:
are we writing objective C now because I am totally for it /jk